service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocmosisCoverDocumentsServiceTest.java

```diff
@@ -55,16 +62,16 @@ void shouldBuildCoverDocumentsWhenAllDataProvided() {
             .put("representativeAddress", "1 Petty France\nSt James's Park\nLondon")
             .build();
 
-        assertThat(generalLetterData).isEqualTo(expectedMap);
+        assertThat(coverDocData.toMap(mapper)).isEqualTo(expectedMap);
     }
 
     @Test
     void shouldDefaultNullFamilyManCaseNumberToEmptyString() {
         Long ccdCaseNumber = 1234123412341234L;
-        Map<String, Object> generalLetterData = documentsService.buildCoverDocumentsData(null,
+        DocmosisCoverDoc coverDoc = documentsService.buildCoverDocumentsData(null,
```

Comment:
full words

Reply:
let's extract null to a variable NULL_FAMILY_MAN_NUMBER or something, a little clearer what and why I think